swagger.yaml

```diff
@@ -4248,3 +4248,22 @@ components:
         description:
           type: string
           description: Optional description of the attachment
+
+    LocalizedName:
+      type: array
+      description: List of all localization values available
+      items:
+        $ref: "#/components/schemas/LocalizedValue"
+
+    LocalizedValue:
+      type: object
+      description: Localized value for one language
+      required:
+        - language
+        - value
+      properties:
+        language:
+          type: string
+          maxLength: 2
```

Comment:
Add a description to this since maxLength 2 means that we are using ISO 639-1 standard